Modules/CalcOffence-3_0.lua

```diff
@@ -2049,11 +2046,45 @@ function calcs.offence(env, actor, activeSkill)
 				output.ShockDurationMod = 1 + skillModList:Sum("INC", cfg, "EnemyShockDuration") / 100 + enemyDB:Sum("INC", nil, "SelfShockDuration") / 100
 				output.ShockEffectMod = skillModList:Sum("INC", cfg, "EnemyShockEffect")
 				if breakdown then
-					t_insert(breakdown.ShockDPS, s_format("For the minimum 5%% Shock to apply for 2 seconds, target must have no more than %d Ailment Threshold.", (((100 + output.ShockEffectMod)^(2.5)) * baseVal) / (100 * m_sqrt(10))))
+					t_insert(breakdown.ShockDPS, s_format("For the minimum 5%% Shock to apply for 2 seconds, target must have no more than %d Ailment Threshold.", ((((100 + output.ShockEffectMod)^(2.5)) * baseVal) / (100 * m_sqrt(10))) * output.ShockDurationMod))
```

Comment:
Duration does not scale the ailment threshold. 

Duration does scale the line `Shock to apply for 2 seconds`